source/module_ri/ABFs_Construct-PCA.cpp

```diff
@@ -15,6 +15,7 @@ namespace ABFs_Construct::PCA
 {
 	void tensor_dsyev(const char jobz, const char uplo, RI::Tensor<double> & a, double*const w, int & info)
 	{
+		info = 0;
```

Comment:
this change looks unreasonable to me, info is a variable that has been passed to this function, why do you want to change its value?